Makefile

```diff
@@ -294,8 +313,19 @@ install-crds:
 
 .PHONY: install-resources
 install-resources:
-	@echo creating namespaces
-	kubectl create ns $(WATCH_NAMESPACE)
+	# creating namespaces
+	-kubectl create ns $(WATCH_NAMESPACE)
+	-kubectl create ns $(KIND_NAMESPACE)
+	# deploying roles and service account
+	kubectl apply -k deploy/rbac
+	kubectl apply -f deploy/manager/service-account.yaml -n $(KIND_NAMESPACE)
+
+.PHONY: kind-ensure-sa
+kind-ensure-sa:
+	@KUBECONFIG_TOKEN="$$(kubectl config view --raw -o jsonpath='{.users[].user.token}')"; \
+	KUBECONFIG_USER="$$(echo "$${KUBECONFIG_TOKEN}" | jq -rR 'split(".") | .[1] | select(. != null) | @base64d | fromjson | .sub')"; \
+	echo "Kubeconfig user detected from token: $${KUBECONFIG_USER}"; \
+	[ "$${KUBECONFIG_USER}" = "system:serviceaccount:open-cluster-management-agent-addon:config-policy-controller" ]
```

Comment:
So ultimately, it just fails if the user isn't what was expected?

Reply:
Yes--it was a last-minute sanity check (slash me playing around with the JWT for fun). We can probably take it out if it seems a bit too much.